api/src/main/java/org/ednovo/gooru/application/util/context/AppContext.java

```
@@ -0,0 +1,25 @@
+package org.ednovo.gooru.application.util.context;
+
+import org.springframework.context.ApplicationContext;
+
+
+/**
+ * @author ashish
+ * The class which is going to hold application context and provide
+ * it to other components on demand.
+ * The setup of context is enabled by AppContextProvider
+ * @see AppContextProvider
+ */
+public class AppContext {
+	
+	public static ApplicationContext getCtx() {
+		return ctx;
+	}
+
+	public static void setCtx(ApplicationContext ctx) {
```

@am-gooru 
another thing to check or throw an exception is if ctx is already set then we should throw an exception unless we are anticipating that context might change at run-time by dynamic bean loading?